CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 ### Fixed
 - Not working json format message.
+### Changed
```

Comment:
Need a blank line before the title

Reply:
Plus, a `Security` section would be more appropriate with fixed issues on the list.